### PR TITLE
Previous attempts

### PR DESCRIPTION
**Pull request by brickandmortar**
I retitled this section to be clearer, and took a step back to give a bigger picture of the trend of tokenized real estate, but also the basic theory of how it's structured. I also incorporated all of dfmerins language, but sprinkled it throughout so it made sense.

### CODE REVIEW

File: whitepaper.md [hunk 1]
```diff
@@ -36,9 +36,17 @@ Attempts to radically rethink the City, like Google's Sidewalk Labs in Toronto h
 
 The current city is technology enhanced, but it is not technology defined. Initial considerations are things like real estate, tax collection and allocation, and voting. What Nakamoto and Buterin did for value and compute, we propose to do for the phyiscal world.
 
-### Previous Examples
+### Fractionalized Ownership
```
Comment (scottfits) [inline whitepaper.md:39]: I propose we call it Fractional Ownership everywhere

File: whitepaper.md [hunk 2]
```diff
@@ -36,9 +36,17 @@ Attempts to radically rethink the City, like Google's Sidewalk Labs in Toronto h
...
+
+Tokenization is fractional ownership represented on a blockchain, which increases liquidity, transparency, and accessibility of properties and their profits, and has been [a dream of members of the Ethereum community since it’s earliest days](https://blog.slock.it/decentralized-sharing-economy-to-be-revealed-at-leading-blockchain-conference-f419f15beb7f?gi=828f6c1a9ee1). Real T is one of the most successful companies to do this so far and has tokenized residential investment properties and sold fractions to individuals all over the world. Profits from these properties are also delivered on chain, increasing the speed in which investors receive cash flow from their investment.
```
Comment (scottfits) [inline whitepaper.md:43]: I think you should call it RealT and link to it

File: whitepaper.md [hunk 3]
```diff
@@ -36,9 +36,17 @@ Attempts to radically rethink the City, like Google's Sidewalk Labs in Toronto h
...
+Currently, real estate is largely recorded offline and transactions require intermediaries, large sums of capital, and extensive legal paperwork. Fractionalized ownership is a method of dividing real estate, or any asset, into pieces or shares so that it can have multiple owners. Timeshares and companies like Fundrise and Roofstock are in essence ways of providing ownership of a single asset to many people. These versions however come with many restrictions regarding their use, transferability, duration of investment, and limits of legal ownership.
```
Comment (scottfits) [inline whitepaper.md:41]: real estate ownership is largely recorded offline

File: whitepaper.md (hunk 2 above)
Comment (brickandmortar) [reply whitepaper.md]: Done

File: whitepaper.md (hunk 3 above)
Comment (brickandmortar) [reply whitepaper.md:41]: that's how it should read now

File: whitepaper.md (hunk 1 above)
Comment (brickandmortar) [reply whitepaper.md:39]: I think that should be fixed now too.

**Pull request merged**